src/domain/chainContext.ts

```diff
@@ -77,9 +77,9 @@ export class ChainContext {
    */
   public async warmUp(oneShot?: boolean) {
     const { provider, chainId } = this;
+    const log = logger.getLogger(`chainContext:warmUp:${chainId}`);
```

Comment:
The chain will be relevant for any log, but only be visible in this logger (other loggers will loose this info).

If we stick to one network per instance, simplifies our logging so we don't need to include the chain id.
Since it still supports multiple RPC nodes, its fine to stay, but as i mentioned, i don't think is a good practice (unless you overcomplicate the other loggers to include the chainId, among other changes that this code would require to safely run on multi-network setup)

Reply:
As discussed, will retain multi-chain capabilities (for a dappnode package). Loggers are injecting the `chainId`.